diff --git a/drivers/iommu/Kconfig b/drivers/iommu/Kconfig
index 217898b9a805..356e79a2eaa6 100644
--- a/drivers/iommu/Kconfig
+++ b/drivers/iommu/Kconfig
@@ -502,7 +502,6 @@ config QCOM_IOMMU
 	# Note: iommu drivers cannot (yet?) be built as modules
 	bool "Qualcomm IOMMU Support"
 	depends on ARCH_QCOM || (COMPILE_TEST && !GENERIC_ATOMIC64)
-	depends on QCOM_SCM=y
 	select IOMMU_API
 	select IOMMU_DMA
 	select IOMMU_IO_PGTABLE_LPAE
