diff --git a/arch/arm64/boot/dts/qcom/msm8916-motorola-harpia.dts b/arch/arm64/boot/dts/qcom/msm8916-motorola-harpia.dts
index ec589b729e42..c6b28d216b7c 100644
--- a/arch/arm64/boot/dts/qcom/msm8916-motorola-harpia.dts
+++ b/arch/arm64/boot/dts/qcom/msm8916-motorola-harpia.dts
@@ -520,7 +520,7 @@ pinconf {
 		};
 	};
 
-	blsp1_uart1_default: blsp1_uart1_default {
+	blsp1_uart1_default {
 		pinmux {
 			function = "blsp_uart1";
 			pins = "gpio0", "gpio1";
@@ -532,7 +532,7 @@ pinconf {
 		};
 	};
 
-	blsp1_uart1_sleep: blsp1_uart1_sleep {
+	blsp1_uart1_sleep {
 		pinmux {
 			function = "gpio";
 			pins = "gpio0", "gpio1";
diff --git a/arch/arm64/boot/dts/qcom/msm8916-motorola-surnia.dts b/arch/arm64/boot/dts/qcom/msm8916-motorola-surnia.dts
index e33e6f858bf2..b163ebb48179 100644
--- a/arch/arm64/boot/dts/qcom/msm8916-motorola-surnia.dts
+++ b/arch/arm64/boot/dts/qcom/msm8916-motorola-surnia.dts
@@ -375,7 +375,7 @@ pinconf {
 		};
 	};
 
-	blsp1_uart1_default: blsp1_uart1_default {
+	blsp1_uart1_default {
 		pinmux {
 			function = "blsp_uart1";
 			pins = "gpio0", "gpio1";
@@ -387,7 +387,7 @@ pinconf {
 		};
 	};
 
-	blsp1_uart1_sleep: blsp1_uart1_sleep {
+	blsp1_uart1_sleep {
 		pinmux {
 			function = "gpio";
 			pins = "gpio0", "gpio1";
